### ISSUE TITLE
Normalization of image

### ISSUE THREAD

**Issue by krantiparida** [intel-isl/DPT#13]
I observed that you have used mean = [0.5,0.5,0.5] and std = [0.5, 0.5, 0.5] for normalizing images for all the transformer based models. It is different from the Imagent normalization value. Are these models not pre-trained on Imagenet?

**Comment by ranftlr** [comment]
The ViT backbones were pretrained on ImageNet, but with different normalization than the usual "ImageNet" normalization: the values [0, 255] are simply mapped to [-1, 1]. 

See here: https://github.com/google-research/vision_transformer/blob/a51dbfa231262ffd1e8e52c041324e6de644d99b/vit_jax/input_pipeline.py#L139

The transform with mean = [0.5, 0.5, 0.5] and std = [0.5, 0.5, 0.5] together with the prenormalization to [0, 1] has the same effect.
